src/containers/TemplateBuilder/template/Form/ElementConfig.tsx

```diff
@@ -253,6 +253,10 @@ const ElementConfig: React.FC<ElementConfigProps> = ({ element, onClose }) => {
                 options={[
                   { value: IsReviewableStatus.Always, text: 'Always' },
                   { value: IsReviewableStatus.Never, text: 'Never' },
+                  {
+                    value: IsReviewableStatus.OptionalIfNoResponse,
+                    text: 'Optional (if no application response)',
+                  },
```

Comment:
Aff everything here is not localised. But that's ok - work for the future

Reply:
Ya, not much in the template builder is localised!